Fix TYPE_CHECKING not imported

minwhoo opened the pull request:
# What does this PR do?

Fixes omitted import of TYPE_CHECKING in xlm_prophetnet model code's __init__.py.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

Commit:
Fix TYPE_CHECKING not imported

Commit:
Fix TYPE_CHECKING not imported

nreimers committed:
Replace // operator with / operator + long() (#13013)

patrickvonplaten committed:
[Flax] Correct pt to flax conversion if from base to head (#13006)

* finish PR

* add tests

* correct tests

* finish

* correct other flax tests

* better naming

* correct naming

* finish

* apply sylvains suggestions

patrickvonplaten committed:
[Flax T5] Speed up t5 training (#13012)

* fix_torch_device_generate_test

* remove @

* update

* up

* fix

* remove f-stings

* correct readme

* up

Co-authored-by: Patrick von Platen <patrick@huggingface.co>

sgugger committed:
[WIP] Disentangle auto modules from other modeling files (#13023)

* Initial work

* All auto models

* All tf auto models

* All flax auto models

* Tokenizers

* Add feature extractors

* Fix typos

* Fix other typo

* Use the right config

* Remove old mapping names and update logic in AutoTokenizer

* Update check_table

* Fix copies and check_repo script

* Fix last test

* Add back name

* clean up

* Update template

* Update template

* Forgot a )

* Use alternative to fixup

* Fix TF model template

* Address review comments

* Address review comments

* Style

sgugger commented:
Actually the whole init needs to be rewritten to be the same as other models, it was somehow missed when we converted all models. Would like to amend your PR in that direction?

michaelbenayoun committed:
FX submodule naming fix (#13016)

Changed the way dynamically inserted submodules are named and the method used to insert them

Authored-by: Michael Benayoun <michael@huggingface.co>

michaelbenayoun committed:
T5 with past ONNX export (#13014)

T5 with past ONNX export, and more explicit past_key_values inputs and outputs names for ONNX model

Authored-by: Michael Benayoun <michael@huggingface.co>

LysandreJik committed:
Put smaller ALBERT model (#13028)

sgugger committed:
Tpu tie weights (#13030)

* Fix tied weights on TPU

* Manually tie weights in no trainer examples

* Fix for test

* One last missing

* Gettning owned by my scripts

* Address review comments

* Fix test

* Fix tests

* Fix reformer tests

lewtun committed:
Use min version for huggingface-hub dependency (#12961)

* Use min version for huggingface-hub dependency

* Update dependency version table

patrickvonplaten committed:
Update README.md

abhishekkrthakur committed:
tfhub.de -> tfhub.dev (#12565)

patrickvonplaten committed:
[Flax] Refactor gpt2 & bert example docs (#13024)

* fix_torch_device_generate_test

* remove @

* improve docs for clm

* speed-ups

* correct t5 example as well

* push final touches

* Update examples/flax/language-modeling/README.md

* correct docs for mlm

* Update examples/flax/language-modeling/README.md

Co-authored-by: Patrick von Platen <patrick@huggingface.co>

Commit:
Fix lazy import of xlm_prophetnet

Commit:
Merge branch 'fix-type-checking-not-imported' of github.com:minwhoo/transformers into fix-type-checking-not-imported

sgugger commented:
Your rebase has introduced many file changes in the diff that make the PR unreadable. Once you're satisfied with your branch, could you close this PR and open a new one? This should make the diff better.

minwhoo commented:
Sure thing.
